tensorflow/contrib/layers/python/layers/layers.py

```diff
@@ -104,44 +106,44 @@ def avg_pool2d(inputs,
                data_format=DATA_FORMAT_NHWC,
                outputs_collections=None,
                scope=None):
-  """Adds a 2D average pooling op.
```

Comment:
These large format-only changes make it really difficult for me to review this change. Can you revert all formatting changes and send only the functional ones?